src/reconstruction/MatrixModel.jl

```diff
@@ -351,7 +351,7 @@ end
     remove_reactions!(
         model,
         [
-            ridx for ridx in 1:n_reactions(model) if
+            ridx for ridx = 1:n_reactions(model) if
```

Comment:
why is the `in` changed to a `=`, but the edit in `src/types/misc/gecko.jl` is the opposite?

Reply:
JuliaFormatter now puts = before ran:ges and `in` before everything non-rangeish